pkg/infrastructure/openstack/clusterapi/clusterapi.go

```diff
@@ -33,7 +33,7 @@ func (p Provider) Name() string {
 
 // BootstrapHasPublicIP indicates that an ExternalIP is not
 // required in the machine ready checks.
-func (Provider) BootstrapHasPublicIP() bool { return false }
+func (Provider) BootstrapHasPublicIP() bool { return true }
```

Comment:
We can't set it to `true`, this will cause a deadlock as the hook that creates the FIP for the bootstrap node is called after the function that waits for the bootstrap node to have an external IP.